src/Magento/FunctionalTestingFramework/DataGenerator/Objects/OperationElement.php

```diff
@@ -64,7 +64,7 @@ public function __construct($key, $value, $type, $required, $nestedElements = []
         $this->value = $value;
         $this->type = $type;
         $this->nestedElements = $nestedElements;
-        if ($required) {
+        if (filter_var($required, FILTER_VALIDATE_BOOLEAN)) {
```

Comment:
If we're implementing `filter_var` aren't we just better off assigning `$this->required = filter_var(....);`?
The `required` attribute in the schema is of `type="xs:boolean"` so we can just assign the string to the converted boolean value.